homeassistant/components/mpd/media_player.py

```diff
@@ -188,6 +196,20 @@ def media_duration(self):
         # Time does not exist for streams
         return self._currentsong.get("time")
 
+    @property
+    def media_position(self):
+        """Position of current playing media in seconds.
+
+        This is returned as part of the mpd status rather than in the details
+        of the current song.
+        """
+        return self._status.get("time")
```

Comment:
We can use `self._media_position` here.

Reply:
Sorry for the delay! That has been updated and tested locally.